app/views/users/show.html.erb

```diff
@@ -17,4 +17,6 @@
 <button type="button" class="btn">
   <%= link_to "See All Posts", user_posts_path(@user) %>
 </button>
+
+<button type="button" class="btn"><%= link_to "Create a post", new_user_post_path(@user)  %></button>
```

Comment:
- [ ] I can't see any users displayed in your page even after creating users in rails console so I recommend to fix your views to let the user be able to see all of your view correctly 🤩✨🔥💯


![image](https://user-images.githubusercontent.com/58553711/222934693-f53b18f3-48f9-4457-a91b-6d3c4e0f2989.png)

Reply:
![users](https://user-images.githubusercontent.com/60324033/222935683-0bf01d2d-3e3b-4040-9046-773e3ca0e00e.PNG)